### DISCUSSION TITLE
Low Link Quality

### DISCUSSION THREAD

**Discussion by Boogaard53** [Koenkk/zigbee2mqtt#8417] [General]
I have a cc2538 with a cc2592 antenna, directly into my RPI's USB port running HA. The system works. But when my devices are a few feet away from my hub with a desk between the hub and devices, the link quality is never above 20. This means half of the time my smart buttons presses don't come trough. Let alone if I move them further away with a wall inbetween

**Comment by Koenkk** [comment]
> I have a cc2538 with a cc2592 antenna, directly into my RPI's USB port running HA.

Check https://www.zigbee2mqtt.io/how_tos/how_to_improve_network_range_and_stability.html#connect-the-adapter-using-an-usb-extension-cable (usb extension cable)

**Reply by Boogaard53** [reply]
I already have an usb extenson cable attached. I am on channel 11 and I cant find out if another channel (and what channel) would improve the quality

**Comment by lux73** [comment]
wifi channel != zigbee channel -> https://www.metageek.com/training/resources/zigbee-wifi-coexistence.html

in my Environment zigbee Channel 25 works best, but this is extremly dependent from your Location (how much other & which Wifi Channels are reserverd, if theres other Zigbee Equipment and so on...)

you need to try out which Zigbee Channel works best for you!
